benchmarks/setup/default/starter.yaml

```diff
@@ -30,11 +30,11 @@ spec:
             value: "warn"
         resources:
           limits:
-            cpu: 2
-            memory: 2Gi
+            cpu: 250m
```

Comment:
:question: Shouldn't we give at least one core? I expect this config to only work well with a low rate? Or is the expected usage now that each starter has a low rate but you increase the number of starters instead?

Reply:
Tbh I dont think that we should configure it here for all use cases, if we want to increase the rate than we can also increase the resource if it is really needed, but I don't think that it is necessary. In most case we use the same rate. That said 2 was just to much in my opinion, but sure we can increase it to 0.5 or 1, but I wouldn't go higher.